.gitignore

```diff
@@ -12,3 +12,9 @@ project/plugins/project/
 # Vagrant
 .vagrant
 VERSION
+
+# IDE and editors
+.idea
+
+# Testing Scala sheets
+src/test/scala/com.snowplowanalytics.snowplow.scalatracker/*.sc
```

Comment:
Can you ignore these files locally instead of in the .gitignore? That way the .gitignore file only references files generated by the project itself, rather than files incidentally created by various people working on the repo.